tests/test_gallery.py

```diff
@@ -191,6 +191,18 @@ def test_albums_sort(settings):
     a.sort_subdirs('title')
     assert [im.title for im in a.albums] == list(reversed(titles))
 
+    orders = [d.meta['order'] for d in a.albums]
+    orders.sort()
+    settings['albums_sort_reverse'] = False
+    a = Album('dir1', settings, album['subdirs'], album['medias'], gal)
+    a.sort_subdirs('meta.order')
+    assert [d.meta['order'] for d in a.albums] == orders
```

Comment:
`d.meta['order'][0]` is maybe safer, I don't know if and how py.test tests recursively list of lists.